src/stavesection.ts

```diff
@@ -3,72 +3,52 @@
 
 import { Stave } from './stave';
 import { StaveModifier } from './stavemodifier';
-import { TextFormatter } from './textformatter';
 import { Category } from './typeguard';
 
 export class StaveSection extends StaveModifier {
   static get CATEGORY(): string {
     return Category.StaveSection;
   }
 
-  protected section: string;
-  protected shiftX: number;
-  protected shiftY: number;
   protected drawRect: boolean;
 
-  constructor(section: string, x: number, shiftY: number, drawRect = true) {
+  constructor(section: string, x: number, yShift: number, drawRect = true) {
```

Comment:
Interesting. Some classes use shiftY and other classes use yShift.

Should we standardize? Which one is better? Do we also use deltaX, dX or something similar?

Reply:
I am standarizing to `yShift` as already used in the base class `Element`